src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/SavingMethodsTestEnvImp.java

```diff
@@ -0,0 +1,52 @@
+package com.greenfoxacademy.opal.kalendaryo.kalendaryo.service;
+
+import com.google.api.client.util.Base64;
+import com.greenfoxacademy.opal.kalendaryo.kalendaryo.model.entity.AuthModel;
+import com.greenfoxacademy.opal.kalendaryo.kalendaryo.model.entity.MergedCalendar;
+import com.greenfoxacademy.opal.kalendaryo.kalendaryo.repository.AuthModelRepository;
+import com.greenfoxacademy.opal.kalendaryo.kalendaryo.repository.MergedCalendarRepository;
+import com.greenfoxacademy.opal.kalendaryo.kalendaryo.repository.UserModelRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.security.SecureRandom;
+
+@Component
+@Profile("test")
+public class SavingMethodsTestEnvImp implements SavingMethods {
+
+    @Autowired
+    AuthModelRepository authModelRepository;
+
+    @Autowired
+    UserModelRepository userModelRepository;
+
+    @Autowired
+    MergedCalendarRepository mergedCalendarRepository;
+
+    @Override
+    public void saveAuthModel(AuthModel authModel) {
+        if (authModelRepository.findByEmail(authModel.getEmail()) == null) {
```

Comment:
@vajdalil awesome! make this whole separation structure one level lower with only one authorize method